freeipa/src/main/java/com/sequenceiq/freeipa/controller/UserV1Controller.java

```diff
@@ -98,6 +102,19 @@ public SyncOperationStatus getSyncOperationStatus(@NotNull String operationId) {
                 operationService.getOperationForAccountIdAndOperationId(accountId, operationId));
     }
 
+    @Override
+    public String getKeytab(@NotNull String environmentCrn, String targetUserCrn) {
+        String callerUserCrn = checkUserCrn();
+        LOGGER.debug("getKeytab() request for environmentCrn={}, callerUserCrn={}, targetUserCrn={}", environmentCrn, callerUserCrn, targetUserCrn);
+        // TODO For now, we ignore the optional targetUserCrn and only allow retrieving keytab for the calling user. When
+        //      we later enable this, we need to make sure that the calling has the necessary authorization / right to get
+        //      keytab for another user.
+        if (targetUserCrn != null) {
+            throw new BadRequestException("Retrieving a keytab for another user is not yet supported");
+        }
```

Comment:
We might take of that `targetUserCrn` from the API and add it when we support it, as it won't be a breaking change. If we keep this way I think we should not throw an exception when target and caller crn are the same.

Reply:
Good point, I've updated to not throw when target is same.